operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -675,7 +675,7 @@ Step createDomainUpPlan(DomainPresenceInfo info) {
 
     return Step.chain(
           createDomainUpInitialStep(info),
-          ConfigMapHelper.readExistingSituConfigMap(info.getNamespace(), info.getDomainUid()),
+          ConfigMapHelper.readExistingDomainConfigMap(info.getNamespace(), info.getDomainUid()),
```

Comment:
This is a confusing rename IMO.  

We have at least four config maps:   'Scripts' configmap deployed per namespace, Introspector Output per domain, Optional Custom Situational Config Input per domain, and Optional Model Input per domain.   

This change renames situational to 'Domain' which is a bit confusing because (a) both Situational and Model are both domain scoped, and (b) the namespaced cm is called 'weblogic-domain-cm'.

Reply:
Suggested nomenclature:   Model, Situational, Introspector, Scripts ???

Reply:
@rjeberhard noted that the config map in question started with just situation config, but now can includes other elements. It's worth bringing up again, though, if it's confusing.

Reply:
There are input config maps, but this is the one generated per-domain config map, correct? The changed name here seems okay to me.

Reply:
It's not OK by me - i find it quite confusing.  The 'script' config map is literally named 'weblogic-domain-cm' but the domain cm is _not_ the 'script' config map.   The domain config map is the introspector config map.  I find this all very confusing. Why not simply call it the introspector config map?  It's contents mirror the contents of the introspector output?  And the name of the config map in k8s has the keyword 'introspectsor' right in it.

Reply:
Looks like this was fixed.